src/pages/Home/SearchSongs/SearchSongs.js

```diff
@@ -5,67 +5,75 @@ import { useIntl } from 'react-intl';
 import { setQueue } from '@context/playing';
 import Input from '@common/Input';
 import { addSong } from '@services/api';
-import { search } from '@services/spotify';
+import { search, topTracks } from '@services/spotify';
 import { SongSearchConstants } from '@utils/constants';
-import { translate } from '@utils';
 
 import SongResult from './SongResult';
 
 import './SearchSongs.scss';
 
 const SearchSongs = () => {
   const intl = useIntl();
+  const [expanded, setExpanded] = useState(false);
   const [offset, setOffset] = useState(0);
   const [song, setSong] = useState('');
   const [searchTimeout, setSearchTimeout] = useState(null);
   const [searchResults, setSearchResults] = useState([]);
 
-  // TODO: Change with custom names
   const user = useSelector(state => state.me.displayName);
   const dispatch = useDispatch();
 
   const searchSongs = async (query, newOffset = offset) => {
     const result = await search(query, newOffset);
+    setExpanded(true);
     setSearchResults(result);
     setSearchTimeout(null);
   };
 
+  const favouriteSongs = async (newOffset = offset) => {
```

Comment:
I'd suggest to use verbs for method naming. For example, `getFavouriteSongs` here and `getTopTracks` in the line below.

Reply:
Agree, done